package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Update notifications for your CLI app",
   "license": "BSD-2-Clause",
   "repository": "yeoman/update-notifier",
+  "bugs": "https://github.com/yeoman/update-notifier/issues",
```

Comment:
This should also not have been part of the PR... It's out of scope and not needed as npm infers it from the repo field.